tests/python/unittest/test_operator.py

```diff
@@ -6990,6 +6990,20 @@ def test_float16_min_max():
     assert np.finfo('float16').max == mx.nd.max(a).asscalar()
 
 
+@with_seed()
+def test_zero_sized_min_max():
+    def min():
+        a = mx.nd.zeros(shape=(5, 0))
```

Comment:
It should have already failed here in the infer-shape function of `zeros`. If you want to use 0 as zero dim size, please turn numpy compatibility by adding a decorator to the test function so that it will fail in `ReduceMinMaxAxesShape` when `a.min()` is called.

```python
@with_seed()
@use_np_compat
def test_zero_size_min_max():
```

Reply:
@reminisce You'are right. After #14661, the error I reported will only be triggered in numpy mode.